### ISSUE TITLE
Instructions for website owners: how to adapt iframe to be AMP ready

### ISSUE THREAD

**Issue by mahnunchik** [ampproject/amphtml#27950]
## Describe the new feature or change to an existing feature you'd like to see

I'm adapting my website to be AMP ready. I've faced with the problem that a little amount of services support embedding as amp-iframe https://amp.dev/documentation/components/amp-iframe/

I mean iframe resizing to be true responsive https://amp.dev/documentation/components/amp-iframe/#iframe-resizing

## Describe alternatives you've considered

Could you please write good instructions for **website/service/iframe owners**: how to adapt their iframe to be AMP ready. This is not difficult and this instruction will help developers to implement it in correct way.

## Additional context

It will be easy for me as website owner send this instruction to iframe owner to show what iframe should supports to be responsive on my website.

**Comment by alanorozco** [comment]
Hi @mahnunchik 

I'm not sure I follow. What do you mean by making a vendor service "AMP-ready"?

For iframe resizing, [the link](https://amp.dev/documentation/components/amp-iframe/#iframe-resizing) you shared contains instructions on how to send the resize message. What instructions are missing?

**Comment by mahnunchik** [comment]
Hi @alanorozco 

Yep that instruction above shows us technical details of implementation.

I mean some kind of motivation for `iframe` owners:

1. Existing iframe may be embedded to AMP pages too.
2. Audience growth.
3. It is easy to implement.

It may be part of documentation https://amp.dev/documentation/components/amp-iframe/#iframe-resizing or even post in blog https://blog.amp.dev/

**Comment by alanorozco** [comment]
I see. I guess that would be similar to the [third-party vendors section](https://amp.dev/documentation/components/amp-video-iframe/#third-party-video-vendors) for `amp-video-iframe`.

@CrystalOnScript: Do you think we have an overarching story for 3rd party embeds here? Should we include a similar section on `amp-iframe` or would we like to consolidate somehow?

**Comment by mahnunchik** [comment]
@alanorozco yep, similar enough. Maybe a bit more details about the benefits for vendor.

My use case: I'm website owner and I'm using 3rd party iframe on my website.

I'm writing to iframe vendor: Hi, could you please make your iframe to be AMP ready? Benefits for you and instruction here: [link]

**Comment by CrystalOnScript** [comment]
I think there's definitely room for this type of documentation, or at least a space to link out to it. Do we have current instructions or vendors? How different would it be from whats on `amp-video-iframe`?
